### PR TITLE
Refactor org.instaslice string using the constant

### PR DESCRIPTION
**Pull request by hasanawad94**
This pr refactors the use cases for "org.instaslice" in code to use the defined constant
#21

### CODE REVIEW

**Comment by harche** [comment]
@hasanawad94 thanks for the PR. 

/ok-to-test

/assign @sairameshv

File: internal/controller/instaslice_daemonset_test.go [hunk 1]
```diff
@@ -111,15 +111,15 @@ func TestCleanUp(t *testing.T) {
...
 	assert.NoError(t, errDeletingInstaSlice)
 
 	var updatedNode v1.Node
 	errVerifyingInstaSliceResource := fakeClient.Get(context.Background(), types.NamespacedName{Name: "node-1"}, &updatedNode)
 	assert.NoError(t, errVerifyingInstaSliceResource)
 
-	_, exists := updatedNode.Status.Capacity["org.instaslice/uid-1"]
-	assert.False(t, exists, "resource 'org.instaslice/uid-1' should be deleted from the node's capacity")
+	_, exists := updatedNode.Status.Capacity[orgInstaslicePrefix + "uid-1"]
+	assert.False(t, exists, "resource '%suid-1' should be deleted from the node's capacity", , orgInstaslicePrefix)
```
Comment (sairameshv) [inline internal/controller/instaslice_daemonset_test.go:122]:
```suggestion
	assert.False(t, exists, fmt.Sprintf("resource '%suid-1' should be deleted from the node's capacity", orgInstaslicePrefix))
```

File: test/e2e/e2e_test.go [hunk 2]
```diff
@@ -166,8 +166,8 @@ var _ = Describe("controller", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred(), "Failed to parse pod JSON")
 
 			finalizers, found := podObj["metadata"].(map[string]interface{})["finalizers"].([]interface{})
-			Expect(found).To(BeTrue(), "Pod does not have finalizers")
```
Comment (sairameshv) [inline test/e2e/e2e_test.go:169]: I think this line has been missed

File: test/e2e/e2e_test.go [hunk 3]
```diff
@@ -166,8 +166,8 @@ var _ = Describe("controller", Ordered, func() {
...
-			Expect(finalizers).To(ContainElement("org.instaslice/accelerator"), "Finalizer org.instaslice/accelerator not found on pod")
+			finalizer := orgInstaslicePrefix + "accelerator"
+            Expect(finalizers).To(ContainElement(finalizer), "Finalizer '%s' not found on pod", finalizer)
```
Comment (sairameshv) [inline test/e2e/e2e_test.go:170]: Please check the indentation here

File: test/e2e/e2e_test.go [hunk 4]
```diff
@@ -166,8 +166,8 @@ var _ = Describe("controller", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred(), "Failed to parse pod JSON")
 
 			finalizers, found := podObj["metadata"].(map[string]interface{})["finalizers"].([]interface{})
-			Expect(found).To(BeTrue(), "Pod does not have finalizers")
-			Expect(finalizers).To(ContainElement("org.instaslice/accelerator"), "Finalizer org.instaslice/accelerator not found on pod")
+			finalizer := orgInstaslicePrefix + "accelerator"
```
Comment (sairameshv) [inline test/e2e/e2e_test.go:169]: This constant `orgInstaslicePrefix` has to be defined inside this `test/e2e` package

File: test/e2e/e2e_test.go [hunk 5]
```diff
@@ -477,7 +477,7 @@ var _ = Describe("controller", Ordered, func() {
 				}
 
 				for resource := range nodeResult.Status.Capacity {
-					if strings.HasPrefix(resource, "org.instaslice/") {
+					if strings.HasPrefix(resource, orgInstaslicePrefix) {
```
Comment (sairameshv) [inline test/e2e/e2e_test.go:480]: The constant `orgInstaslicePrefix` has to be defined in this package as a constant.

File: test/e2e/e2e_test.go [hunk 6]
```diff
@@ -503,7 +503,7 @@ var _ = Describe("controller", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred(), "Failed to reset nvidia.com/mig-1g.5gb to zero")
 
 			err = deleteInstasliceResources()
-			Expect(err).NotTo(HaveOccurred(), "Failed to delete org.instaslice/* resources")
+			Expect(err).NotTo(HaveOccurred(), "Failed to delete %s* resources", orgInstaslicePrefix)
```
Comment (sairameshv) [inline test/e2e/e2e_test.go:506]: Once the constant `orgInstaslicePrefix` is defined,
```suggestion
			Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Failed to delete %s* resources", orgInstaslicePrefix))
```

**Comment by hasanawad94** [comment]
Will see about the tests

**Review by sairameshv — CHANGES_REQUESTED** [review]

**Comment by rphillips** [comment]
Adding an internal/utils module with a GenerateInstasliceResourceName(name string) is probably worth it.

**Review by mamy-CS — APPROVED** [review]
/lgtm

**Pull request merged**